packages/ckeditor5-table/src/converters/tableproperties.js

```diff
@@ -14,8 +14,9 @@
  * @param {String} modelElement
  * @param {String} modelAttribute
  * @param {String} styleName
+ * @param {Boolean} [reduceBoxSides=false]
  */
-export function upcastStyleToAttribute( conversion, modelElement, modelAttribute, styleName ) {
+export function upcastStyleToAttribute( conversion, modelElement, modelAttribute, styleName, reduceBoxSides = false ) {
```

Comment:
4 arguments for the function seem to be too many. Let's improve the API a little bit:

```js
upcastStyleToAttribute( conversion, { 
	modelElement: 'modelElement',
	modelAttribute: 'modelAttribute',
	styleName: 'styleName',
	reduceBoxSides: true // false by default (if not specified)
} );
```

The same should we do with the internal helper: `enableProperty()`.

Reply:
:+1: